colcon_core/environment/__init__.py

```diff
@@ -126,8 +126,12 @@ def create_environment_scripts(
     # create file containing the runtime dependencies
     path = prefix_path / get_relative_package_index_path() / pkg.name
     path.parent.mkdir(parents=True, exist_ok=True)
+    dependencies = [dependency.name
+                    for dependency in pkg.dependencies.get('run', set())]
```

Comment:
Nitpick: shorten local variable name to: `dependencies = [d.name for d in pkg.dependencies.get('run', set())]`

In general I try to avoid aligned indentation. Maybe the new code fits in a single line. If not please wrap after the open parenthesis / bracket.